eng/pipelines/templates/steps/analyze.yml

```diff
@@ -37,9 +37,12 @@ steps:
     failOnStderr: true
     workingDirectory: 'sdk/${{parameters.ServiceDirectory}}'
 
-  - script: |
-      echo check source file formatting in $(pwd)
-      gofmt -s -l -d $(find . -name '*.go' -print) >&2
+  - pwsh: |
+      Write-Host "##[command]Check source file formatting in $(pwd)"
+      $files = Get-ChildItem -Path . -Filter *.go
+      if ($files.Count -gt 0) {
+        gofmt -s -l -d $files
```

Comment:
From the [gofmt docs](https://pkg.go.dev/cmd/gofmt) it says "given a directory, it operates on all .go files in that directory, recursively" so I wonder if we need to find the files at all?